Sources/EdgeNetworkHandlers/RequestBuilder.swift

```diff
@@ -26,6 +26,9 @@ class RequestBuilder {
     /// XDM payloads to be attached to the request
     var xdmPayloads: [String: AnyCodable] = [:]
 
+    var sdkConfig: SDKConfig?
+    var configOverrides: [String: AnyCodable]?
```

Comment:
Can you add a description to this property? Such as "Configuration override metadata for Edge Network services."